model/commentsModel.js

```diff
@@ -60,4 +60,25 @@ PostQuery = (username,body,id) =>{
     .then(({rows})=>{
         return rows
     })
+}
+
+exports.selectDeletedCommentByCommentId = (id)=>{
+
+    const checkIdExists = db.query(`
```

Comment:
don't strictly need this separate check - we can see if the delete was successful by checking the rowCount property on the result of the delete query (represents the number of records deleted - 0 if id does not exist and no comment was deleted, 1 if it did exist and was successfully deleted)